buildscripts/hpat-conda-recipe/bld.bat

```diff
@@ -3,3 +3,7 @@ REM set OPENCV_DIR="%LIBRARY_PREFIX%"
 REM set DAALROOT="%LIBRARY_PREFIX%"
 "%PYTHON%" setup.py build install --single-version-externally-managed --record=record.txt
 if errorlevel 1 exit 1
+"%PYTHON%" setup.py build_doc
+if errorlevel 1 exit 1
+"%PYTHON%" setup.py build_devdoc
+if errorlevel 1 exit 1
```

Comment:
Could you please comment these lines out? I mean, do not delete them but comment out for a moment. We will need them later.

Reply:
Yes sure. I'll do that.